docs/src/main/sphinx/release/release-392.md

```diff
@@ -0,0 +1,34 @@
+# Release 392 (27 July 2022)
+
+## General
+
+* Add support for dynamic filtering with task retries. ({issue}`9935`)
+* Improve performance when joining tables on a single `bigint` column. ({issue}`13178`)
+* Fix potential query failure in certain complex queries with multiple joins and
+  aggregations. ({issue}`13315`)
+
+## Delta Lake connector
+
+* Add support for using a randomized location when creating a table, so that
+  future table renames or drops do not interfere with new tables created with
+  the same name. ({issue}`6063`)
+
+## Hive connector
+
+* Add support for short timezone IDs when translating Hive views. For example,
+  `JST` now works as an alias for `Asia/Tokyo`. ({issue}`13179`)
+* Add support for the Avro file format. ({issue}`12125`)
+
+## Iceberg connector
+
+* Add support for hidden `$file_modified_time` columns. ({issue}`13082`)
+* Improve query performance for tables with updated or deleted rows. ({issue}`13092`)
+* Fix failure when inserting into a Parquet table with columns that have
+  quotation marks in their names. ({issue}`13074`)
+
+## MySQL connector
+
+* Change mapping for the MySQL `enum` type to the Trino `varchar` type instead
+  of the `char` type. ({issue}`13303`)
+* Fix failure when reading table statistics while the
+  `INFORMATION_SCHEMA.COLUMN_STATISTICS` table doesn't exist. ({issue}`13323`)
```

Comment:
change to lowercase

Reply:
The actual table name is uppercase in MySQL. I don't disagree to lowercase though. 
https://dev.mysql.com/doc/refman/8.0/en/information-schema.html